app.js

```diff
@@ -3,13 +3,22 @@ const cp = require('child_process');
 const linter = require('./lib/linter');
 const upgrader = require('./lib/upgrader');
 const { stripIndent } = require('common-tags');
+const prompts = require('prompts');
 
 if (argv._[0] === 'reset') {
   cp.execSync('git reset --hard && git clean -df');
 } else if (argv._[0] === 'lint') {
   linter({ argv });
 } else if (argv._[0] === 'upgrade') {
-  upgrader({ argv });
+  confirm()
```

Comment:
Rather than introduce promise chain syntax it would be better to move the "main program" into a "go" async function, and invoke it (without await, since there is no top level await) at top level. This will still cause node to hang around until it's done.

Reply:
By "main program" I mean this if/else... block that does the top level work.